### DISCUSSION TITLE
Leds go SOFT. Xbox HDR not triggering hardware brightness level

### DISCUSSION THREAD

**Discussion by Abraham79** [awawa-dev/HyperHDR#648] [Q&A]
xBox One S & HyperHDR on Pi3 + lolin mini ESP32 + ms2130 + Ezcoo splitter, everything working as expected everywhere but on the xBox. 

SDR is flawless, but as soon as HDR triggers, leds go soft. Only on the xBox. 

PC & Zidoo working, both SDR & HDR. I have tried different HDMI cables, still not working. Does anyone know how to fix?

**Comment by Abraham79** [comment]
I mean, tv triggers HDR without any issues, hyperHDR doesnt. It could be related to HDR10 format compatibility, splitter works (as it does work on TV)

**Comment by awawa-dev** [comment]
I'm not sure if I understand correctly, but seems it's a problem with a splitter. Grabber should be connect to ezcoo HDMI2 port (TV to HDMI1) and maybe you should check/test scaller options that are available for Ezcoo. Also try to disconnect/reconnect Ezcoo usb power supply if you dont have a video in HyperHDR video preview: that's what matter here, not that the LEDs are off (this is only a consequence).

**Reply by Abraham79** [reply]
I think it couldnt  be the splitter, since TV still triggers HDR and TV is also behind the same splitter. Also, that wont explain its working as expected for any other sources. Maybe HyperHDR is not understanding some particularities xBox HDR10 format might have

Anyhow I checked and made sure I have it connected exactly as you described. Splitter modes didnt make a difference either

**Reply by awawa-dev** [reply]
> I think it couldnt be the splitter, since TV still triggers HDR and TV is also behind the same splitter.

This only means that one of the outputs of the HDMI splitter sends the signal to the TV. It says nothing about what is produced at the second output of the HDMI splitter and whether the video signal from that output is acceptable to the USB grabber. That's all.

**Reply by Abraham79** [reply]
Besides, leds are not OFF, just hardware brightness level isnt working, they show correct colors, just not enough brightness (I repeat, xBox only)

**Reply by awawa-dev** [reply]
search ms2130 thread, power sequence could be important as it could be ezcoo hardware problem, and as I said before:  `try to disconnect/reconnect Ezcoo usb power supply` when the problem appears. Do not mess with the grabber hardware brightness! Should be set to 0 for ms2130 (requires restart).

**Reply by Abraham79** [reply]
First of all, I forgot to say THANKYOU!

will try later as soon as I get back home. Already read the ms2130 thread and the issue with the ezcoo splitter

I just got a ms2109 delivered today, also I have a feintech splitter I had discarded because it wasnt downscaling properly (it was everything pink with ms2130). So I’ll try those also. I have nothing to loose since Pi3 doesnt have usb3.0 support

**Reply by awawa-dev** [reply]
👍  Probably the correct power-on sequence should solve the problem (of course, if this is also the case here). However, it would be better if someone notified Ezcoo if there was a path that would allow them to reproduce the problem. I respect Ezcoo because they provide firmware updates, so there's a small chance they could fix it.

**Reply by Abraham79** [reply]
Here to confirm, power-cycling ezcoo splitter does indeed solve the problem. Thank you! 

BUT, 

whenever I exit back to SDR then back again into HDR, it stops working once again, until I repeat the ezcoo power-cycling trick EVERY TIME! 

Will try ms2109 + feintech splitter tomorrow.

Again, thank you for your support!

**Reply by Abraham79** [reply]
Still, its only happening to fail while xBox is involved, while other devices can trigger and exit HDR without any issues

**Reply by Abraham79** [reply]
Pelase notice user “appleimperio” from ms2130 thread is also using xBox while facing the same issues

**Reply by awawa-dev** [reply]
I understand that such a situation may occur, but I very much doubt that Ezcoo support will follow this thread and be able to help. Please contact them directly.

**Reply by Abraham79** [reply]
Swapped capture card to ms2109, now its working on both ezcoo & feintech splitters. 

So yeah, ezcoo splitter is causing this, but just if/while combined with ms2130 & Xbox. Too much for ezcoo team to even care to reproduce the issue (I did contact then though).

Also, I get wrong/weird rainbow colors from ms2109 when idle or while HDMI handshake:

![E1B792D6-CF4F-4B6C-9F01-F5247FA8E1A9](https://github.com/awawa-dev/HyperHDR/assets/17594626/801ae72c-2c9b-4e20-b3a5-52757a137e80)

Is this behavior expected? Can this be fixed to a Black screen or a single color?

Thank you!

**Reply by awawa-dev** [reply]
Ezcoo is likely causing problems with the ms2130 EDID settings, which may be different than the ms2109 EDID. Turn on the 'live' stream in the HyperHDR preview window and you will see a full ms2109 rainbow "no signal" board. You can experiment with automatic/manual signal detection, but I doubt it will cure it completely if it only happens for a few seconds: a longer period is usually required.
